README.md

```diff
@@ -0,0 +1,69 @@
+## Prerequisites
+* Node
+* Yarn / NPM
+* Go
+* Your folder is inside GOPATH
+
+## Pact
+
+```bash
+cd pact
+```
+
+### Run consumer
+
+```bash
+cd consumer
+yarn
+yarn jest
+```
+
+### Run provider
+
+Ensure you've installed pact's [CLI Tools](https://github.com/pact-foundation/pact-ruby-standalone/releases).
+
+```bash
+cd provider
+go test . -v
+```
+
+## Apidocs
+
+```bash
+cd swagger
+```
+
+The API definition is under apidocs/specs/openapi.yaml
+
+### Run consumer
+
+Be sure to have Prism installed: `curl https://raw.githubusercontent.com/stoplightio/prism/2.x/install.sh | sh`
+
+```bash
+cd consumer
```

Comment:
since `prism` does not need a specific folder to run, maybe we don't need to do `cd consumer` here, and afterwards we can just do `prism mock -s apidocs/specs/openapi.yaml`